src/components/KuadrantPoliciesPage.tsx

```diff
@@ -0,0 +1,422 @@
+import * as React from 'react';
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { Dropdown, DropdownItem, DropdownList, MenuToggle, MenuToggleElement, Alert, AlertGroup, Title} from '@patternfly/react-core';
+import { sortable } from '@patternfly/react-table';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import { useK8sWatchResource, K8sResourceCommon, ResourceLink, useActiveNamespace, HorizontalNav, useListPageFilter,
+  ListPageBody,
+  ListPageCreate,
+  ListPageFilter,
+  VirtualizedTable,
+  TableData,
+  RowProps,
+  TableColumn,
+  NamespaceBar,
+  Timestamp,
+  useActivePerspective,
+ } from '@openshift-console/dynamic-plugin-sdk';
+import './kuadrant.css';
+
+interface Resource {
+  name: string;
+  gvk: {
+    group: string;
+    version: string;
+    kind: string;
+  };
+}
+
+interface ExtendedK8sResourceCommon extends K8sResourceCommon {
+  status?: {
+    addresses?: { value: string }[];
+  };
+}
+
+const statusConditionsAsString = (obj: any) => {
```

Comment:
not urgent to move or genericise, but this might be useful elsewhere. mentally noting so I know where to nab.